Makefile

```diff
@@ -1,40 +1,51 @@
 SHELL = /bin/sh
-PROJECT = graze/silex-trailing-slash-handler
+IMAGE ?= graze/php-alpine
 
-.PHONY: install clean help
-.PHONY: test test-unit test-matrix
+DOCKER_RUN := docker run --rm -it -v $$(pwd):/srv -w /srv ${IMAGE}
+
+.PHONY: install update update-lowest help
+.PHONY: test test-unit test-matrix test-matrix-lowest
 
 .SILENT: help
 
-install: ## Download the depenedencies then build the image :rocket:.
-	make 'composer-install --optimize-autoloader --ignore-platform-reqs'
-	docker build --tag ${PROJECT}:latest .
+install: ## Download the dependencies then build the image :rocket:.
```

Comment:
In `make help`, this shows up as this:

```
install                 Download the dependencies then build the image              rocket  .
```

(Scroll across to see the problem.)

Given this is intended for command line display, I don't think the `:rocket:` is useful.

Reply:
used a 'real' rocket: 🚀